src/main/java/frc/robot/subsystems/swerve/swervecontainer/real/RealGyroConstants.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.swerve.swervecontainer.real;
+
+import com.ctre.phoenix6.configs.Pigeon2Configuration;
+import frc.robot.constants.IDs;
+import frc.robot.subsystems.swerve.gyro.pigeon2.Pigeon2GyroConfigObject;
+
+public class RealGyroConstants {
+
+    private static final Pigeon2Configuration PIGEON_2_CONFIGURATION = new Pigeon2Configuration();
+    static {
+        PIGEON_2_CONFIGURATION.MountPose.MountPoseRoll = 180;
+        PIGEON_2_CONFIGURATION.MountPose.MountPosePitch = 0;
+        PIGEON_2_CONFIGURATION.MountPose.MountPoseYaw = 0;
+    }
```

Comment:
static blocks are pretty ugly,
consider using 
`new Pigeon2Configuration().withMountPose(
            new MountPoseConfigs().withMountPoseRoll(180).withMountPosePitch(0).withMountPoseYaw(0)
    );`
or something easier to read.

Reply:
![image](https://github.com/user-attachments/assets/93f7abcd-cdc3-4a36-aab2-bd593992c553)

![image](https://github.com/user-attachments/assets/ce586f8e-15e2-4ccd-bcfa-648e4cfe4a86)


If we use builder it needs to be consistent, in cases like this i think its harder to read